api/all/src/main/java/io/opentelemetry/api/metrics/DoubleUpDownCounter.java

```diff
@@ -9,7 +9,7 @@
 import io.opentelemetry.context.Context;
 import javax.annotation.concurrent.ThreadSafe;
 
-/** An up-down-counter instrument that records {@code double} values. */
+/** An up down counter instrument that records {@code double} values. */
```

Comment:
```suggestion
/** An UpDownCounter instrument that records {@code double} values. */
```
(that's how the spec has it I think)